worker/pkg/workflows/datasync/activities/gen-benthos-configs/processors.go

```diff
@@ -435,7 +435,7 @@ func computeMutationFunction(col *mgmtv1alpha1.JobMapping, colInfo *sqlmanager_s
 
 	switch col.Transformer.Source {
```

Comment:
Main question I have here:

If these now all have presence detection, using the Getters removes that check right, so it will always be the system default?

Seems ok for bools, empty arrays, etc, but for min/max it is kinda weird, no?

```
minValue := col.Transformer.Config.GetGenerateFloat64Config().GetMin()
maxValue := col.Transformer.Config.GetGenerateFloat64Config().GetMax()
```

If none of these are provided, 0, 0 will be passed in to the transformer.
Is that desired, or would we now need to conditionally build the blobl method to handle omitting those values so that it falls back to the transformer defined default value?


```
generateFullName := col.Transformer.Config.GetGenerateCountryConfig().GetGenerateFullName()
return fmt.Sprintf(`generate_country(generate_full_name:%t)`, generateFullName), nil
```
Would result in an empty generateFullName being passed in.

Reply:
If you wanna chat more tomorrow after standup.